docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
Agreed. As the Swarm introduction is not really straightforward, we removed the `$` from *template* commands (containing `<...>`) as suggested by @moxiegirl to avoid confusion and make a clear distinction between *template* commands and *runnable* ones.